exchange/utils.go

```diff
@@ -17,6 +17,28 @@ import (
 	"github.com/prebid/prebid-server/privacy/lmt"
 )
 
+func bidderToPrebidSChains(req *openrtb_ext.ExtRequest) map[string]*openrtb_ext.ExtRequestPrebidSChainSChain {
+	bidderToSChains := make(map[string]*openrtb_ext.ExtRequestPrebidSChainSChain)
+
+	if req.Prebid.SChains == nil || len(req.Prebid.SChains) == 0 {
+		return bidderToSChains
+	}
+
+	for _, schainWrapper := range req.Prebid.SChains {
+		if schainWrapper != nil && len(schainWrapper.Bidders) > 0 {
+			for _, bidder := range schainWrapper.Bidders {
+				if _, present := bidderToSChains[bidder]; present {
+					delete(bidderToSChains, bidder)
+					continue
```

Comment:
I think it would make more sense here (easier to read IMHO) to use an if / else here than the continue keyword.